.github/workflows/merge-check.yml

```diff
@@ -30,8 +30,37 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
+
+      - name: Cache CVE database Files
+        uses: actions/cache@69d9d449aced6a2ede0bc19182fadc3a0a42d2b0 # v3
```

Comment:
Should we move to Gradle Build Action? I believe they might be caching the whole `~/.gradle` automatically (to verify)

Reply:
It would be different, because of restore-keys. Anyway we probably could, but I would still merge this as is.

Reply:
Also the CVE wouldn't be cached, because it's just  a random folder that happens to be in ~/.gradle
https://github.com/gradle/gradle-build-action#caching